### PR TITLE
feat: add commandline inputs for import process

### PR DESCRIPTION
**Pull request by smomin**

### CODE REVIEW

File: POSTINSTALL.md [hunk 1]
```diff
@@ -42,18 +42,20 @@ Run the import process using `npx`.
...
-        ALGOLIA_API_KEY=${param:ALGOLIA_API_KEY}\
-        ALGOLIA_INDEX_NAME=${param:ALGOLIA_INDEX_NAME}\
-        COLLECTION_PATH=${param:COLLECTION_PATH}\
-        FIELDS=${param:FIELDS}\
-        TRANSFORM_FUNCTION=${param:TRANSFORM_FUNCTION}\
-        GOOGLE_APPLICATION_CREDENTIALS=</path/to/service/account/key>\
-        npx firestore-algolia-search
+      npx firestore-algolia-search
+      ```
+      Below are the questions that will be asked?
```
Comment (Haroenv) [inline POSTINSTALL.md:47]:
```suggestion
      Below are the questions that will be asked
```

File: functions/cli.js [hunk 2]
```diff
@@ -1,4 +1,64 @@
 #!/usr/bin/env node
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
 
-require('./lib/import/index');
+rl.question(`\nWARNING: The back fill process will index your entire collection which will impact your Search Operation Quota.  Please visit https://www.algolia.com/doc/faq/accounts-billing/how-algolia-count-records-and-operation/ for more details.  Do you want to continue? (y/N): `, function(answer) {
```
Comment (Haroenv) [inline functions/cli.js:5]: would you be comfortable using the Promises API https://nodejs.org/api/readline.html#promises-api

**Review by Haroenv — COMMENTED** [review]
it's good that those questions are asked as well, easier to avoid errors

File: functions/cli.js (hunk 2 above)
Comment (smomin) [reply functions/cli.js:5]: I would be comfortable with promises but its only available on Node v. 18.x.  The older versions are not have the readline set up with Promises at this time.  I will convert it over later.

**Pull request merged**